hpotter/plugins/plugin_handler.py

```diff
@@ -4,7 +4,7 @@
 import docker
 import re
 
-from hpotter.plugins import plugin
+from hpotter.plugins.plugin import *
```

Comment:
This line should be from hpotter.plugins.plugin import Plugin.
Try to avoid using the wildcard on imports